.github/renovate.json

```
@@ -2,14 +2,19 @@
   "$schema": "https://docs.renovatebot.com/renovate-schema.json",
   "extends": ["github>bitwarden/renovate-config:non-pinned"],
   "separateMajorMinor": true,
-  "enabledManagers": ["cargo", "github-actions", "npm", "nuget"],
+  "enabledManagers": ["cargo", "dockerfile", "github-actions", "npm", "nuget"],
```

```suggestion
  "enabledManagers": ["cargo", "docker", "github-actions", "npm", "nuget"],
```

The readme says this should be `docker`? https://docs.renovatebot.com/modules/datasource/docker/